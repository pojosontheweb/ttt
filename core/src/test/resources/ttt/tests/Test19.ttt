<%@ page import="java.util.List" %>
<%!
    List<Number> numbers;
%>
this is generic...