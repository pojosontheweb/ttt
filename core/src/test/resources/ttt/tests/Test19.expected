package ttt.tests;

import java.util.List;

public class Test19 extends com.pojosontheweb.ttt.Template {

	private final List<Number> numbers;

	/**
	 * Creates an instance of this template.
	 *
	 */
	public Test19(List<Number> numbers) {
		this.numbers = numbers;
	}

	@Override
	public void doRender(com.pojosontheweb.ttt.TttWriter out) throws Exception {
		write(out, "this is generic..." );
	}
}
